test/upgradeability/AdminUpgradeabilityProxy.test.js

```diff
@@ -246,43 +246,43 @@ contract('OwnedUpgradeabilityProxy', ([_, owner, anotherAccount]) => {
     })
   })
 
-  describe('transferOwnership', function () {
-    describe('when the new proposed owner is not the zero address', function () {
-      const newOwner = anotherAccount
+  describe('changeAdmin', function () {
+    describe('when the new proposed admin is not the zero address', function () {
+      const newAdmin = anotherAccount
 
-      describe('when the sender is the owner', function () {
+      describe('when the sender is the admin', function () {
         beforeEach('transferring', async function () {
-          const { logs } = await this.proxy.transferProxyOwnership(newOwner, { from: owner })
+          const { logs } = await this.proxy.changeAdmin(newAdmin, { from: admin })
           this.logs = logs
         })
 
-        it('assigns new proxy owner', async function () {
-          const proxyOwner = await this.proxy.proxyOwner({ from: newOwner })
-          assert.equal(proxyOwner, anotherAccount)
+        it('assigns new proxy admin', async function () {
+          const admin = await this.proxy.admin({ from: newAdmin })
```

Comment:
You are overriding the `admin` variable you are already declaring in the global list of accounts injected by truffle. Please use `proxyAdmin` or so to make it clearer.